pages/search.vue

```diff
@@ -92,9 +95,9 @@ const playItem = (item: TrackModel, group: IDiscoverableGroup) => {
         </button>
       </div>
       <div class="border-t border-label-4 p-4">
-        <div v-show="activeTab === 'All'">
+        <div>
           <ol class="w-full divide-y divide-label-separator">
-            <template v-for="item in results?.items" :key="item.id">
+            <li v-for="item in results?.items" :key="item.id">
```

Comment:
Was there a reason why you had to change this to an `li` tag? As of what I see, there's no reason to have this, since all children here have the tag `li`...